tests/layer_tests/tensorflow_tests/test_tf_Slice.py

```diff
@@ -20,17 +31,45 @@ def create_slice_net(self, input_shape, input_type, begin_value, size_value):
 
         ref_net = None
         return tf_net, ref_net
+    
+    def create_complex_slice_net(self, input_shape, input_type, begin_value, size_value):
+        tf.compat.v1.reset_default_graph()
+        with tf.compat.v1.Session() as sess:
+            real_input = tf.compat.v1.placeholder(input_type, input_shape, 'real_input')
+            imag_input = tf.compat.v1.placeholder(input_type, input_shape, 'imag_input')
+            complex = tf.raw_ops.Complex(real= real_input, imag = imag_input)
+            slice = tf.raw_ops.Slice(complex, begin = begin_value, size = size_value)
+            real = tf.raw_ops.Real(input =slice)
+            imag = tf.raw_ops.Imag(input = slice)
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        ref_net = None
+        return tf_net, ref_net
+
+            
 
     test_data_basic = [
-        dict(input_shape=[6], input_type=tf.float32, begin_value=[2], size_value=[2]),
-        dict(input_shape=[2, 5, 3], input_type=tf.int32, begin_value=[0, 1, 0], size_value=[-1, 1, -1]),
-        dict(input_shape=[10, 5, 1, 5], input_type=tf.float32, begin_value=[5, 1, 0, 3], size_value=[2, 4, -1, -1]),
-    ]
+    dict(input_shape=[6], input_type=tf.float32, begin_value=[2], size_value=[2]),
+    dict(input_shape=[2, 5, 3], input_type=tf.int32, begin_value=[0, 1, 0], size_value=[-1, 1, -1]),
+    dict(input_shape=[10, 5, 1, 5], input_type=tf.float32, begin_value=[5, 1, 0, 3], size_value=[2, 4, -1, -1]),
+    dict(input_shape=[3, 3], input_type=tf.float32, begin_value=[1, 0], size_value=[1, 3]),
+    dict(input_shape=[4, 4, 4], input_type=tf.float32, begin_value=[0, 1, 2], size_value=[-1, 2, 1]),
+    dict(input_shape=[5, 5, 5, 5], input_type=tf.float32, begin_value=[2, 2, 2, 2], size_value=[2, 2, 2, 2]),
+]
+
+
 
     @pytest.mark.parametrize("params", test_data_basic)
     @pytest.mark.precommit_tf_fe
     @pytest.mark.nightly
     def test_slice_basic(self, params, ie_device, precision, ir_version, temp_dir, use_legacy_frontend):
-        self._test(*self.create_slice_net(**params),
+        self._test(self.create_slice_net(*params),
                    ie_device, precision, ir_version, temp_dir=temp_dir,
                    use_legacy_frontend=use_legacy_frontend)
+    
+
+    def test_slice_complex(self, params, ie_device, precision, ir_version, temp_dir, use_legacy_frontend):
+        self._test(self.create_complex_slice_net(*params),
+                   ie_device, precision, ir_version, temp_dir=temp_dir,
+                   use_legacy_frontend=use_legacy_frontend)
```

Comment:
please don't change existing layer tests and add separate test suite for complex type. See other PRs with examples of layer tests for complex type.

Reply:
Done